interface/forms/newpatient/common.php

```diff
@@ -86,16 +86,35 @@ function refreshIssue(issue, title) {
   s.options[s.options.length] = new Option(title, issue, true, true);
  }
 
+ <?php
```

Comment:
Yes. The logic in the validateUsingPageRules method is written for a regular form submission where there is a submit input button. However this form is different than the others, since there is a special function in javascript (called 'saveClicked') which acts onclick, and if we want the  new validation to trigger, we need to overwrite the function. (When a submit is made with plain javascript [like in saveClicked func] the 'onsubmit' eventhandler doesn't catch it. Therefore we cant use an eventhandler, but must overwrite the func in case new validation is on).
So we thought it makes more sense to add this logic in common.php (and not to the general method in validate_core) because this logic is relevant only to this specific form. Do you agree?